semp/metricDesc.go

```diff
@@ -477,4 +478,15 @@ var MetricDesc = map[string]Descriptions{
 		"connection_fast_retransmit":      NewSemDesc("connection_fast_retransmit", NoSempV2Ready, "The number of TCP segments retransmitted due to the receipt of duplicate acknowledgments (‘ACKs’). See RFC 5681 for further details.", variableLabelsVpnClient),
 		"connection_timed_retransmit":     NewSemDesc("connection_timed_retransmit", NoSempV2Ready, "The number of TCP segments re-transmitted due to timeout awaiting an ACK. See RFC 793 for further details.", variableLabelsVpnClient),
 	},
+	"RestConsumerStats": {
+		"http_post_request_messages_sent":                   NewSemDesc("rdp_http_post_request_messages_sent", NoSempV2Ready, "The numer of request messages sent.", variableLabelsRestConsumer),
```

Comment:
Number of what kind of requests send to where?
Please be more specific, to make help text helpfull for non solace broker seniors.

Reply:
I've make the description more clear. Please check if it is ok. thanks.